### ISSUE TITLE
Error for symfony v7

### ISSUE THREAD

**Issue by featuriz** [doctrine-extensions/DoctrineExtensions#2794]
There is a bug in [src/Timestampable/Mapping/Event/Adapter/ORM.php](https://github.com/featuriz/DoctrineExtensions/blob/main/src/Timestampable/Mapping/Event/Adapter/ORM.php) on line 35

```php
return $converter->convertToPHPValue($this->getRawDateValue($mapping), $platform);
```

> $this->getRawDateValue($mapping) 
> Expected type 'array'. 
> Found 'Doctrine\ORM\Mapping\FieldMapping'

**Comment by featuriz** [comment]
Solution: https://github.com/featuriz/DoctrineExtensions/commit/8e7135195984b6d150f4142df408046e1070c37e

**Comment by mbabker** [comment]
This has nothing to do with Symfony 7.

You are trying to use ORM 3.x which [this package is not yet compatible with](https://github.com/doctrine-extensions/DoctrineExtensions/issues/2708).  Please downgrade to ORM 2.x.

**Comment by featuriz** [comment]
Yes @mbabker . I have found and mentioned the file that causing the problem. I request the maintainer to update ORM (upcoming).
I'm testing on this. Not using in production (up to now).

**Comment by mbabker** [comment]
I just looked at some old PRs today and this particular compat issue is fixed in this package's 3.15 release via #2765, so this issue can be closed out.  There are still some other things to fix, but this specific item isn't one of them.

**Comment by featuriz** [comment]
Okay @mbabker I'm closing this
